Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -54,6 +54,7 @@ $ErrorActionPreference = 'Stop'
 $deploymentOperation = 'Deployment'
 
```

Comment:
please remove all instances of this from this file

Write-Telemetry "DTLSDK_Error" $deploymentResponse.DeploymentSummary"

Reply:
Refer https://github.com/Microsoft/vsts-tasks/pull/6889

Reply:
Done. @asranja  Please review it.